src/main/java/startsteps/ECommerceShop/exceptions/GlobalExceptionHandler.java

```diff
@@ -59,4 +57,14 @@ public ResponseEntity<Object> handleRuntimeException(RuntimeException exception)
                 .status(HttpStatus.INTERNAL_SERVER_ERROR)
                 .body(errorResponse);
     }
+    @ExceptionHandler(EmptyCartException.class)
+    public ResponseEntity<Object> handleEmptyCartException(EmptyCartException exception) {
+        ErrorResponse errorResponse = new ErrorResponse(
+                HttpStatus.NOT_FOUND.value(),
```

Comment:
I wouldn't return 404 if the cart is empty. Maybe 200 with empty data or 204 with no data

Reply:
deleted exception, created new response with message, empty list and total price = 0